planetiler-basemap/src/main/java/com/onthegomap/planetiler/basemap/layers/Transportation.java

```diff
@@ -434,7 +435,7 @@ int getMinzoom(Tables.OsmHighwayLinestring element, String highwayClass) {
   private boolean isPierPolygon(Tables.OsmHighwayLinestring element) {
     if ("pier".equals(element.manMade())) {
       try {
-        if (element.source().worldGeometry() instanceof LineString lineString && lineString.isClosed()) {
+        if (element.source().worldGeometry()instanceof LineString lineString && lineString.isClosed()) {
```

Comment:
There's a bug in the current version of the eclipse formatter that removes the space before `instanceof` - should be fixed when they upgrade to 4.22 https://github.com/diffplug/spotless/issues/1059